<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>916</width>
      <height>222</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(&quot;./src/gui/images/title.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>302</width>
      <height>88</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(&quot;./src/gui/images/input.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>250</y>
      <width>180</width>
      <height>68</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(&quot;./src/gui/images/ucs.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>250</y>
      <width>180</width>
      <height>68</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(&quot;./src/gui/images/astar.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>540</y>
      <width>502</width>
      <height>88</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(&quot;./src/gui/images/start.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>370</y>
      <width>502</width>
      <height>88</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(&quot;./src/gui/images/graph.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>680</y>
      <width>501</width>
      <height>88</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(&quot;./src/gui/images/final.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>340</y>
      <width>1691</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(&quot;./src/gui/images/output.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>370</y>
      <width>991</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>1530</x>
      <y>240</y>
      <width>264</width>
      <height>88</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(&quot;./src/gui/images/opengmap.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>240</y>
      <width>923</width>
      <height>89</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(&quot;./src/gui/images/algorithm.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>810</y>
      <width>501</width>
      <height>88</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(&quot;./src/gui/images/cost.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>830</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>321</width>
      <height>51</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(&quot;./src/gui/images/wallpaper.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>381</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>560</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>700</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>-60</x>
      <y>940</y>
      <width>2191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(&quot;./src/gui/images/path.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>960</y>
      <width>1251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>470</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(&quot;./src/gui/images/show.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>pushButton</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>widget</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>label_3</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>label_8</zorder>
   <zorder>label_10</zorder>
   <zorder>comboBox_2</zorder>
   <zorder>lineEdit_3</zorder>
   <zorder>lineEdit</zorder>
   <zorder>label_9</zorder>
   <zorder>label_11</zorder>
   <zorder>pushButton_5</zorder>
  </widget>
 </widget>
 <resources>
  <include location="gui.qrc"/>
 </resources>
 <connections/>
</ui>
